zanata-adapter-xliff/src/main/java/org/zanata/adapter/xliff/XliffWriter.java

```diff
@@ -216,30 +221,12 @@ public static void writeFile(File file, Resource doc, String locale,
             // end xliff tag
             writer.writeEndDocument();
             writer.flush();
+            writer.close();
```

Comment:
Okay, but can we make it Closeable, perhaps by subclassing? 

If we can't or don't want to do that, and if it is important to call `close()` (why else did they provide the method?), we should use try-finally.

Whatever the current Saxon implementation might be (the source won't download for me), the javadoc for `XMLStreamWriter.close()` in the interface says:

    /**
     * Close this writer and free any resources associated with the
     * writer.  This must not close the underlying output stream.
     * @throws XMLStreamException
     */

Besides, it's always a good habit to close things.